frontend/packages/ux-editor/src/components/TextResourceEdit.test.tsx

```diff
@@ -131,26 +155,35 @@ additionalValue });
   });
 });
 
-const render = async (resources: ITextResources = {}, editId?: string) => {
-
+const render = async (
+  resources: ITextResources = {},
+  editId?: string,
+  previewIframeRef?: RefObject<HTMLIFrameElement>,
+) => {
   const textResources: ITextResourcesState = {
     ...textResourcesMock,
-    currentEditId: editId
+    currentEditId: editId,
   };
 
   const appData: IAppDataState = {
     ...appDataMock,
-    textResources
+    textResources,
   };
 
-  const { result } = renderHookWithMockStore({ appData }, {
-    getTextLanguages: () => Promise.resolve(['nb', 'nn', 'en']),
-    getTextResources: (_o, _a, lang) => Promise.resolve<ITextResourcesWithLanguage>({
-      language: lang,
-      resources: resources[lang] || []
-    }),
-  })(() => useTextResourcesQuery(org, app)).renderHookResult;
+  const { result } = renderHookWithMockStore(
+    { appData },
+    {
+      getTextLanguages: () => Promise.resolve(['nb', 'nn', 'en']),
+      getTextResources: (_o, _a, lang) =>
+        Promise.resolve<ITextResourcesWithLanguage>({
+          language: lang,
+          resources: resources[lang] || [],
+        }),
+    },
+  )(() => useTextResourcesQuery(org, app)).renderHookResult;
   await waitFor(() => expect(result.current.isSuccess).toBe(true));
 
-  return renderWithMockStore({ appData })(<TextResourceEdit/>);
+  const reload = previewIframeRef?.current?.contentWindow?.location?.reload;
+  reload?.();
```

Comment:
Er det noen grunn til at du har nullsjekk på alle nivåer? Jeg tror ikke det er nødvendig.
```suggestion
  previewIframeRef.current?.contentWindow.location.reload();
```